src/D2L.Bmx/BmxPaths.cs

```diff
@@ -1,11 +1,14 @@
 namespace D2L.Bmx;
 
 internal static class BmxPaths {
-	public static readonly string USER_HOME_DIR = Environment.GetFolderPath( Environment.SpecialFolder.UserProfile );
-	public static readonly string BMX_DIR = Path.Join( USER_HOME_DIR, ".bmx" );
-	public static readonly string CACHE_DIR = Path.Join( BMX_DIR, "\\cache" );
```

Comment:
the point of `Path.Join` is that we don't need to worry about path separators.
including a backslash also results in incorrect behaviour on POSIX systems, where backslash can be part of a valid directory name.
@alex-fang0